Extensions.targets

```diff
@@ -15,18 +24,19 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="_AddMetadataToExtensions" BeforeTargets="RestoreExtensions;PublishUpgradeAssistantExtensions">
+  <Target Name="_AddMetadataToExtensions"
+          BeforeTargets="RestoreExtensions;PublishUpgradeAssistantExtensions">
     <!-- Add the relative directory the extension will be added to -->
```

Comment:
some weird formatting going on here, I don't see any other changes in this line.

Reply:
It's intentional to align the attributes to read things better.